.circleci/config.yml

```diff
@@ -133,6 +133,43 @@ jobs:
             make migrate
             make check
 
+  # ---- Front-end jobs ----
+  build-front:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/fun/src/frontend
+    steps:
+      - checkout:
+          path: ~/fun
+      - restore_cache:
+          keys:
+            - v1-front-dependencies-{{ checksum "yarn.lock" }}
+            - v1-front-dependencies-
+      - run:
+          name: Install frontend dependencies
+          command: make install
+      - run:
+          name: Build application styles
+          command: make build-sass
+      - save_cache:
+          paths:
+            - ./node_modules
+          key: v1-front-dependencies-{{ checksum "yarn.lock" }}
+
+  lint-prettier:
```

Comment:
This task is not executed in the `workflow` section

Reply:
This is why it does not fail :grin: